packages/mux-player/src/styles.css

```diff
@@ -2,6 +2,7 @@
   /* default aspect ratio, overwrite w/ e.g. style="aspect-ratio: 16 / 10" */
   aspect-ratio: 16 / 9;
   display: block;
+  width: 100%;
```

Comment:
non-blocking: not sure that we want the player to automatically grow to 100% of the width of its parent by default, but we can iterate on that.

Reply:
should we have a min-width instead, then?

Reply:
I kinda do prefer it expands like a block element, who's creating a non responsive player nowadays?